src/main/java/com/graphhopper/converter/api/GHEntry.java

```diff
@@ -14,6 +14,7 @@
     private String osmType;
 
     private Point point;
+    private Extent extent;
```

Comment:
I think we should not create a separate class and instead use List<Double> directly here. See problem https://github.com/graphhopper/geocoder-converter/issues/38#issuecomment-377686474